README.md

```diff
@@ -1,3 +1,10 @@
+HOW TO RUN:
+
+example: npm test (results without stack trace)
+
+	npm test 1 (results with stack trace) 
+
+
```

Comment:
Looks like other cli options are done like so. Shouldn't we stick to this convention?
>   --no-ansi        Disable ANSI formatting
>   --no-progress    Disable progress output during tests
Perhaps `--no-stack`?